Tests/FluentAssertions.Specs/Exceptions/AsyncFunctionExceptionAssertionSpecs.cs

```diff
@@ -85,7 +85,9 @@ public async Task When_async_method_throws_an_empty_AggregateException_it_should
         }
 
         [Collection("UIFacts")]
+#pragma warning disable SA1601 // Partial elements should be documented
```

Comment:
As we don't care about documentation in test projects, you can just exclude SA1601 by adding it to 
https://github.com/fluentassertions/fluentassertions/blob/develop/Tests/FluentAssertions.Specs/.editorconfig
and
https://github.com/fluentassertions/fluentassertions/blob/develop/Tests/FluentAssertions.Equivalency.Specs/.editorconfig

Reply:
Ah.. cool ;)